File: ejercicios hechos/Java/estructuras alternativas/act1.java
/*Programa: act1.java
 * 
 *Propósito: Algoritmo que pida dos números e indique si el primero es mayor que el segundo o no.
 * 
 *Autor: Jose Manuel Serrano Palomo.
 * 
 *Fecha: 08/11/2019
 *
 *Análisis:
 * Debemos pedir un número al usuario y tenemos que comprobar si es par o impar
 *
 *Diseño:
 * 1. Leer el número
 * 2. Si el resto de n/2 es igual a 0 el numero es par
 * 3. sino es impar
 * Variables: n sera el numero que introduzca el usuario
*/
import java.util.Scanner;
public class act1 {
    public static void main(String[] args) {
        System.out.println("Programa para comprobar si es par o impar");
        System.out.println("-----------------------------------------");
        Scanner s = new Scanner(System.in);
        int n;
        System.out.print("Introduce un número: ");
        n = s.nextInt();
        if (n%2==0) {
        System.out.println("Es Par");
        }   else {
        System.out.println("Es impar");
        }
    }
}

File: ejercicios terminados/Python/estructuras alternativas/act1.py
#Programa: act1.py
#Propósito: Algoritmo que pida dos números e indique si el primero es mayor que el segundo o no.
#Autor: Jose Manuel Serrano Palomo.
#Fecha: 17/10/2019
#
#Variables a usar:
# n1 y n2 serán los número que vamos a pedir al usuario
# 
#Algoritmo:
# LEER n1,n2
# Comparar n1 y n2
# Escribir si n1 es mayor o menor que n2

print("Comparación entre dos números")
print("-------------------------------\n")

#Leemos los datos

n1 = int(input("Introduce el primer número: "))
n2 = int(input("Introduce el segundo número: "))

#Comparamos y mostramos el resultado en pantalla

if n1 > n2:
    print(f"{n1} es mayor que {n2}")
else:
    print(f"{n1} es menor que {n2}")
